### DISCUSSION TITLE
PAGFile通过远端URL异步加载时，能否补充下缓存参数？

### DISCUSSION THREAD

**Discussion by zz6123816** [Tencent/libpag#2631] [Ideas] [answered]
<img width="499" alt="image" src="https://github.com/user-attachments/assets/6be916db-029f-4ed4-a3aa-b97b21a9bc15" />
RT，能否增加“是否为缓存”和“缓存路径”的参数回调， 在PAGFile不支持网络URL加载之前，我们是手动封装了一套，而且接入业务会针对 是否缓存、缓存路径 等内容做些业务自己的处理。 这里打算切成PAGFile自身加载模式，但当前回调信息缺少缓存相关内容，所以暂时只能还是使用自己封装的下载模型

**Comment by domchen** [comment, marked as answer]
业务已经封装了就推荐用业务自定义的，PAG这里的网络加载只提供最基本的实现。
